e2e/deep-matcher.ts

```diff
@@ -34,3 +34,15 @@ export function deepDataMatcher(data: any, pattern: any, path = 'ROOT'): string[
     }
     return errors;
 }
+
+export function isValidEthereumAddress(str: string) {
+    return str && str.startsWith('0x') && str.length == '0x5cd0D270C30EDa5ADa6b45a5289AFF1D425759b3'.length;
```

Comment:
length is 42 and it might be worth checking that its `/0x[0-9a-fA-F]/` ?

Reply:
I think length of an example address is more readable than a constant :)

And IMO not worth checking. We have unit tests for these things. Here it's just to check the general integration.